maxim/logger/langchain/tracer.py

```
@@ -256,7 +262,28 @@ def on_llm_start(
+                    "container_id": generation_container.id,
+                    "generation_container": generation_container,
+                    "evaluators": (
+                        self.eval_config.get(metadata["langgraph_node"], [])
+                        if self.eval_config
+                        else []
+                    ),
+                    "node_name": metadata["langgraph_node"],
+                    "input": last_input_message,
+                }
+                self.to_be_evaluated_container_store.set(
+                    str(generation_container.id), eval_data, DEFAULT_TIMEOUT
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)